_posts/2022-09-15-homebrew-maintainer-projects-update.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Maintainer Projects
+author: SMillerDev
+---
+
+Homebrew’s Project Leadership Committee has green-lit two paid projects by our maintainers this year and since both have hit some milestones recently we’d love to give you, our sponsors and users, an update on the progress.
+
+## API Project
+Since version [3.6.0](./2022-09-07-homebrew-3.6.0.md) of Homebrew, we are now choosing to advertise the `HOMEBREW_INSTALL_FROM_API` environment variable to the general public. A lot of work was done by @Rylan12 to make the experience more stable and the API install method is bound to bring a big performance improvement to those who dare enable it.
+
+The goal of the `HOMEBREW_INSTALL_FROM_API` environment variable is to allow Homebrew users to install formulae and casks without needing to have Homebrew/homebrew-core and/or Homebrew/homebrew-cask tapped. This is intended to save space on the user’s machine and also to make commands like `brew update` much faster.
+
+Now that the variable is public, we’re going into a period of bug hunting. When the users stop finding problems and edge-cases for the API to handle, we will be making this the default for all users so everyone can enjoy a fast Homebrew experience.
+
+## Ephemeral CI Runners
+Since July 18th the Intel based CI runners are now ephemeral and the basis has been laid for the same features for Apple Silicon based runners. While Homebrew users might not notice more than sleeping a little easier, I can tell you that the maintainers are very happy with this improvement and the accompanying dashboard.
```

Comment:
```suggestion
Since July 18th the Intel-based CI runners are now ephemeral and the foundation has been laid for applying the same features to Apple Silicon based runners. While Homebrew users might not notice more than sleeping a little easier thanks to bottling runs being more reliable, I can tell you that the maintainers are very happy with this improvement and the accompanying dashboard.
```
(I doubt the average Homebrew user would actually notice at all, but I've added a potential reason)

Reply:
Yeah, I was going for them sleeping easier because of the better security.